app/lib/bk/compat/parsers/bitrise/steps.rb

```diff
@@ -71,6 +71,62 @@ def translate_git_tag(inputs)
         def translate_script(inputs)
           inputs['content']
         end
+
+        def translate_github_status(inputs)
+          return '# Invalid github-status step configuration!' unless required_github_status_inputs(inputs)
```

Comment:
this should not do a `return` (that would just add the comment to the step's command) but a `raise` instead to signal a huge error in translation of the step